analysis/quantifier-comparisons/01-import-quant-data.Rmd

```diff
@@ -0,0 +1,235 @@
+---
+title: "SCRNAseq quantification comparisons: Data Import"
+author: "Joshua Shapiro for CCDL"
+output: 
+  html_notebook:
+    toc: true
+    toc_float: true
+---
+
+This notebook contains code to import and do preliminary processing single cell expression data, quantified by Alevin, Kallisto, and Cellranger. 
+Alevin samples were quantified with and without decoy sequences. 
+The imported data is stored as SingleCellExperiment objects, which are then saved as an `.rds` file for later analysis.
+
+## Setup
+
+### Load Libraries for import
+```{r setup}
```

Comment:
There are a few places in this notebook where spacing that could be improved for readability. I'm going to be nitpicky about it because my eyes but also acknowledge that it is nitpicking.

```suggestion
### Load Libraries for import

```{r setup}
```